release/files_common

```diff
@@ -531,17 +531,13 @@ opendbc/can/common.h
 opendbc/can/common.pxd
 opendbc/can/common_dbc.h
 opendbc/can/dbc.cc
-opendbc/can/dbc.py
-opendbc/can/dbc_template.cc
 opendbc/can/packer.cc
 opendbc/can/packer.py
 opendbc/can/packer_pyx.pyx
 opendbc/can/parser.cc
 opendbc/can/parser.py
 opendbc/can/parser_pyx.pyx
-opendbc/can/process_dbc.py
-opendbc/can/dbc_out/.gitkeep
-opendbc/can/dbc_out/.gitignore
+opendbc/can/dbc_pyx.pyx
 
 opendbc/chrysler_pacifica_2017_hybrid.dbc
```

Comment:
we should remove the DBC whitelist and include all the ones in the root. we didn't do this before because they got built into libdbc which was huge